### PR TITLE
Subheader created

### PR DESCRIPTION
**Pull request by AFaioes**
Subheader created, but needs to be updated...

### CODE REVIEW

File: src/components/common/SubHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <img src={Pen} alt="Pen logo" />
+            Categoria Produtos 4 
+            </Button>
+              </Dropdown>
+            </li>
+              
+          </ul>
+      </div><ul className="subheader-menu-item">
+              
+          </ul></>
```
Comment (Diogo1080) [inline src/components/common/SubHeader/index.tsx:66]: This should be cleaned up.

Maybe some indentations?

**Review by Diogo1080 — CHANGES_REQUESTED** [review]
Remember to clean the code up. 🧹

**Review by Diogo1080 — APPROVED** [review]
All cleaned up 👍

**Review by mariajoaomorais — APPROVED** [review]
ok!

**Pull request merged**